print('raising')
raise ValueError('bad value')
